core/storage/classifier/gae_models.py

```diff
@@ -95,19 +100,19 @@ def create(
                 data used by the classifier.
 
         Returns:
-            ID of the new ClassifierModel entry.
+            ID of the new ClassifierDataModel entry.
 
         Raises:
             Exception: A model with the same ID already exists.
         """
 
         instance_id = cls._generate_id(exp_id)
```

Comment:
So, wait, the classifier_id is generated based on job_request_id and a random hash? instead of exp_id?